sdk/src/utils/request.ts

```diff
@@ -22,7 +22,8 @@ export const request = async <Output = any>(
 	return fetcher(url, options).then((response) => {
 		return extractData(response).catch((reason) => {
 			const errors = typeof reason === 'object' && 'errors' in reason ? reason.errors : reason;
-			return Promise.reject({ errors, response });
+			const data = typeof reason === 'object' && 'data' in reason ? reason.data : null;
```

Comment:
The above has already been there, but just noticed that we should adjust these two lines to at least ensure the value is not `null` since the `in` operator would fail in that case, so e.g.
```js
			const errors = reason && typeof reason === 'object' && 'errors' in reason ? reason.errors : reason;
			const data = reason && typeof reason === 'object' && 'data' in reason ? reason.data : null;
```

Reply:
Good point! I've added that in